### ISSUE TITLE
How to predict prototype label without ground truth semantic label

### ISSUE THREAD

**Issue by Anna0509** [twke18/SPML#6]
Hi,
In  "pyscripts/inference/prototype.py" (line 202), the ground truth  "semantic label" is required to predict "prototype_labels". How can I get the prototype_labels if I only have partial (e.g. 10 pixels) ground truth "semantic label"?

**Comment by twke18** [comment]
The script assumes that you have per-pixel semantic segmentation labels. For inference under weakly-supervised settings, there are two steps: 1) using the learned features to propagate softmax classifier predictions ([this line](https://github.com/twke18/SPML/blob/f09e4c30ecf2030d42ac70b2c35e7fdeee9bf468/bashscripts/voc12/train_spml_point.sh#L210-L223)), 2) re-train (only) the softmax classifier ([this line](https://github.com/twke18/SPML/blob/f09e4c30ecf2030d42ac70b2c35e7fdeee9bf468/bashscripts/voc12/train_spml_point.sh#L239-L243)) for predict the semantic segmentation results ([this line](https://github.com/twke18/SPML/blob/f09e4c30ecf2030d42ac70b2c35e7fdeee9bf468/bashscripts/voc12/train_spml_point.sh#L249-L260)).

**Comment by jyhjinghwang** [comment]
@Anna0509 
Have you successfully done predictions w/o any labeled set with the instructions that @twke18 provided?
